http/services/httpService.js

```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+
+const config = {
+    baseURL: "http://localhost:8000/api",
+};
+
+class HttpService {
+    constructor() {
+        const client = require("axios").default.create(config);
+        client.interceptors.request.use(this.setAuthorizationHeader);
+
+        this.client = client;
+    }
+
+    setAuthorizationHeader(config) {
+        const token = localStorage.getItem("token");
+        config.headers.Authorization = `Bearer ${token}`;
+        return config;
+    }
```

Comment:
attachHeaders(headers) {
    Object.assign(this.client.defaults.headers, headers);
  }

moze ovako ? a authorization header metodu bi drzao u auth servisu jer je vezano za auth